### PR TITLE
TG CRYOOOOOOO

### PR DESCRIPTION
**Pull request by ghost**
capture

Yes they bob.

:cl:
spriteadd: Awesome cryocells
/:cl:

### CODE REVIEW

**Comment by Partheo** [comment]
I like our own more personally.

**Comment by AshCorr** [comment]
Can we use our cryotube sprites instead of TG ones?

**Comment by ShadowDeath6** [comment]
cool thing but I dislike tg's cryo pods

**Comment by ghost** [comment]
give me the proper sprites and sure (our current sprites wouldn't support this, this requires transparent sprites!)

**Comment by AshCorr** [comment]
> give me the proper sprites and sure (our current sprites wouldn't support this, this requires transparent sprites!)

Edit our current sprites to be transparent.

**Comment by ghost** [comment]
@AshCorr i ain't no spriter

**Comment by Super3222** [comment]
rip

**Comment by ghost** [comment]
these look much nicer

**Comment by Xantam-13** [comment]
I don't like the transparency. Imagine revs, the captain or another head in the cryo who has been beaten, right now you'd probably walk past and think nothing of it. With this cryo, you'd see the head and eject him and beat him more.

**Comment by X-TheDark** [comment]
I think these look neat. If transparency becomes an issue, it can be fixed at a later date.

**Comment by ghost** [comment]
@05rhardy that'd just delay the beating until they're ejected, where a rev would either morphine/kill them or just toolbox them to death

File: config/admins.txt [hunk 1]
```diff
@@ -57,4 +57,3 @@ Hadrin Alucard	=	Retired Admin
  
 Yogstation13-Bot	=	Bot
 
-
```
Comment (silicons) [inline config/admins.txt]: newlines ree

**Comment by X-TheDark** [comment]
I hope you don't mind me messing with your PR directly to fix some issues caused by our timer system being outdated.

**Comment by ghost** [comment]
it's fine

**Comment by ShadowDeath6** [comment]
rip nice looking sprites

**Pull request merged**